src/i18n/ja/strings.txt

```diff
@@ -34,6 +34,7 @@ confirm_delete_ok_button=削除
 library_title=タイトル
 library_body=詳細
 library_tags=タグ
+library_blog=Journal
```

Comment:
Should be translated later, but will confirm... might need to create diff branch on main project